### PR TITLE
2048 Game

### PR DESCRIPTION
**Pull request by SAMRIDHISINHA**
### 🛠️ Fixes Issue (Number)

#677 

I have made a 2048 game using javascript

### ✅ Check List (Check all the applicable boxes) 

- [ ✅  ] My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- [ ✅ ] This PR does not contain plagiarized content.
- [ ✅ ] My Addition/Changes works properly and matches the overall repo pattern.
- [ ✅ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by NOiR-07** [comment]
don't delete any line of code from readme just add yours

File: cards.json [hunk 1]
```diff
@@ -694,5 +694,10 @@
         "name": "YouTube Clone",
         "image": "assets/Images/YouTubeClone/YouTubeClone.png",
         "link": "Public/YouTube_Clone/index.html"
+    },
+    {
+        "name": "2048 Game",
```
Comment (Astrodevil) [inline cards.json:699]: Update it alphabetically.

**Comment by SAMRIDHISINHA** [comment]
@Astrodevil  Please review the changes once. I have updated the README.md and cards.json files.

**Review by SAMRIDHISINHA — COMMENTED** [review]
I have updated the cards.json and README.md files.

**Pull request merged**